tests/solidity/test_open_short.py

```diff
@@ -0,0 +1,199 @@
+"""Test opening a short in hyperdrive"""
+import unittest
+import decimal
+
+import elfpy.agents.agent as agent
+import elfpy.markets.hyperdrive.hyperdrive_market as hyperdrive_market
+import elfpy.pricing_models.hyperdrive as hyperdrive_pm
+import elfpy.types as types
+import elfpy.time as time
+
+
+class TestOpenShort(unittest.TestCase):
+    """
+    Test opening a short in hyperdrive, with the following cases:
+        open a short of 0 size (failure)
+        open a short of extreme size (failure)
+    """
+
+    contribution: float = 500_000_000
+    target_apr: float = 0.05
+    term_length: int = 365
```

Comment:
I like term_length in general, but for now can you keep this as position_duration to keep it consistent with solidity. We can go back this & related variable names when we do the time refactor.